cmd/util/cmd/checkpoint-list-tries/cmd.go

```diff
@@ -29,7 +30,8 @@ func init() {
 func run(*cobra.Command, []string) {
 
 	log.Info().Msgf("loading checkpoint %v", flagCheckpoint)
-	tries, err := wal.LoadCheckpoint(flagCheckpoint, &log.Logger)
+	dir, file := filepath.Split(flagCheckpoint)
+	tries, err := wal.LoadCheckpoint(dir, file, &log.Logger)
```

Comment:
could you break out the LoadCheckpoint changes into a different PR (we can probably get that merge now, but the reviewing the rest of the PR will take longer).